listenbrainz/webserver/decorators.py

```diff
@@ -56,3 +57,18 @@ def decorator(*args, **kwargs):
         return func(*args, **kwargs)
 
     return decorator
+
+
+def web_musicbrainz_needed(func):
+    """
+        This web decorator checks to see if musicbrainz db is online (by having
+        a DB URI) and if not, it redirects to an error page telling the user
+        that the musicbrainz db is offline.
+    """
+    @wraps(func)
+    def decorator(*args, **kwargs):
+        if musicbrainz_db.engine is None:
```

Comment:
This is going to cause problems in local development if people don't have a musicbrainz database - perhaps we need a separate config item in this case?

Reply:
Fixed by adding a separate config variable.